app/api/schema/events.py

```diff
@@ -24,32 +24,6 @@ class Meta:
         self_view_many = 'v1.event_list'
         inflect = dasherize
 
-    @validates_schema(pass_original=True)
-    def validate_date(self, data, original_data):
-        if 'id' in original_data['data']:
-            try:
-                event = Event.query.filter_by(id=original_data['data']['id']).one()
-            except NoResultFound:
-                raise ObjectNotFound({'source': 'data/id'}, "Event id not found")
-
-            if 'starts_at' not in data:
-                data['starts_at'] = event.starts_at
-
-            if 'ends_at' not in data:
-                data['ends_at'] = event.ends_at
```

Comment:
Where is this gone?

Reply:
@iamareebjamal missed it. Added it now. Please check